include/suiryoku/locomotion/process/locomotion.hpp

```diff
@@ -36,6 +36,7 @@ class Locomotion
 public:
   explicit Locomotion(std::shared_ptr<Robot> robot);
 
+  void set_config_name(const std::string & name) {config_name = name;}
```

Comment:
It's better to just make the `config_name` variable public instead of adding a `set_config_name` function.